.github/workflows/elixir_deps_check.yaml

```diff
@@ -36,5 +36,14 @@ jobs:
         env:
           MIX_ENV: emqx-enterprise
           PROFILE: emqx-enterprise
+      - name: produced lock files
+        uses: actions/upload-artifact@v3
+        if: failure()
+        with:
+          name: produced_lock_files
+          path: |
+            mix.lock
+            rebar.lock
+          retention-days: 1
```

Comment:
would it make more sense to merge this into https://github.com/emqx/emqx/pull/11116?

Reply:
since this is a very tiny debugging addition, I'd keep it here to avoid rebasing the 2 PRs and waiting for CI again.  :speak_no_evil: